src/pretix/control/forms/event.py

```diff
@@ -240,6 +240,21 @@ class EventSettingsForm(SettingsForm):
         help_text=_("Require customers to fill in the names of all attendees."),
         required=False
     )
+    attendee_emails_asked = forms.BooleanField(
+        label=_("Ask for attendee e-mails"),
+        help_text=_("Ask for an e-mail address for all tickets which include admission to the event. Important: For "
```

Comment:
Hmmm. I see, but do you have an idea for a new label? "Ask for email addresses of all attendees" isn't much clearer, maybe "Ask for additional attendee e-amil addresses"?